### DISCUSSION TITLE
罕见中文字体出现[F]码如何解决？

### DISCUSSION THREAD

**Discussion by cyzhou1221** [nuaatug/nuaathesis#64] [Q&A] [answered]
如 “彧” 和 “皞” 两个字均出现了乱码，貌似旧版的模板中没有这个问题.

**Comment by yzwduck** [comment, marked as answer]
模板近期没有修改过字体相关的选项，并且我的环境在默认配置下，能正确输出这两个汉字。

GB2312 以外的汉字对字体有不少要求，Linux/macOS 用户的默认中文字体可能不包含这些汉字，建议改用 Source Han Sans/Serif 或者 Windows 自带的字体，需要修改文档类的 `fontset` 选项，或者用 "global.tex" 顶部的方法。

**Reply by cyzhou1221** [reply]
问题已解决！解决方法：
在 Powershell 输入 `tlmgr update --self --all` 进行宏包升级.
